ruby/exercises/Romans.rb

```diff
@@ -0,0 +1,18 @@
+class RomanNum
+
+def self.rom_a_numer( numerito)
+    simb = {1000 => "M", 900 => "CM", 500 => "D", 400 => "CD", 100 => "C", 90 => "XC", 50 => "L", 40 => "XL", 10 => "X", 9 => "IX", 5 => "V", 4 => "IV", 1 => "I"}
```

Comment:
Esto podria ser una constante de la clase
```suggestion
    DEC_ROM_TRANSLATIONS = {
      1000 => "M",
      900 => "CM",
      500 => "D",
      400 => "CD",
      100 => "C",
      90 => "XC",
      50 => "L",
      40 => "XL",
      10 => "X",
      9 => "IX",
      5 => "V",
      4 => "IV",
      1 => "I",
    }
```